src/components/Sketches/components/SketchBox.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import "../../../styles/Sketches.css";
+
+import { Row, Col } from "reactstrap";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+class SketchBox extends React.Component {
+  render() {
```

Comment:
Thoughts on stateless components? We have a couple in the app, but we should be consistent. I'm impartial to either

Reply:
As in the 
```
const Sketchbox = props => {
  return (
     Render equivalent
  )
}
```
style

Reply:
Yeah, I think that'd be reasonable. I have no opinion on stateless vs. normal components, haven't really had to decide on either. Any pros/cons we should look out for?

Reply:
```
However, as of React v16, there are no performance benefits from using stateless functional components over class components. 
```
oof. well there's our answer. It makes it easier if you want to add state later if you have a stateful component so lets stick with this